src/components/sponsor-table/columns.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import { ColumnDef } from "@tanstack/react-table";
+
+export type Payment = {
```

Comment:
Super minor change but once the Reimbursement schema is merged into develop, could you refer to the schema instead? Thanks!